feature-pack/src/main/resources/modules/system/layers/base/com/redhat/gss/redhat-support-lib-java/main/module.xml

```diff
@@ -0,0 +1,23 @@
+<module xmlns="urn:jboss:module:1.0" name="com.redhat.gss.redhat-support-lib-java">
+    
+    <resources>
+        <artifact name="${com.redhat.gss:redhat-support-lib-java}"/>
+    </resources>
+
+    <dependencies>
+        <module name="javax.api"/>
+        <module name="javax.ws.rs.api" services="import"/>
+        <module name="javax.net.ssl" slot='main'/>
+        <module name="org.apache.commons.net" slot='main'/>
+        <module name="org.apache.httpcomponents"/>
+        <module name="org.jboss.staxmapper"/>
+        <module name="org.jboss.as.controller"/>
+        <module name="org.jboss.as.server"/>
+        <module name="org.jboss.modules"/>
+        <module name="org.jboss.msc"/>
```

Comment:
looking at deps of this jar https://github.com/redhataccess/redhat-support-lib-java/blob/master/pom.xml
it looks like you dont need 
```
        <module name="org.jboss.staxmapper"/>
        <module name="org.jboss.as.controller"/>
        <module name="org.jboss.as.server"/>
        <module name="org.jboss.modules"/>
        <module name="org.jboss.msc"/>
```